microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -59,14 +59,10 @@ This release adds improvements related to the following components and concepts.
 [id="microshift-4-17-networking_{context}"]
 === Networking
 
-//[id="microshift-4-17-multus_{context}"]
-//==== Multiple networks capability now available
-
-//[id="microshift-4-17-configure-ingress-router_{context}"]
-//==== Custom configurations for the ingress router are supported
-
-//[id="microshift-4-17-configure-route-admission-policy_{context}"]
-//==== Configuring the route admission policy now available
+[id="microshift-4-17-IPv6_{context}"]
+==== IPv6 single and dual-stack IPv6 networking now available
+With is release, you can run {microshift-short} and your workloads with either IPv6 only or IPv4 and TPv6 dual stack networking protocols. Ingress and egress traffic can be on either IPv4 or IPv6 traffic, mixed or parallel. See xref:../microshift_configuring/microshift-using-config-tools.adoc#microshift-using-config-tools[Configuring IPv6 networking].
```

Comment:
Is there a typo here ? I see TPv6 in the paragraph, and it seems like it should be IPv6.

Reply:
Good catch! Thanks, @dhensel-rh 

@aireilly is this too little for a Vale catch, do you think? I don't see TPv6 anywhere in the repo. Wonder if this can be a suggestion-level event?

Reply:
We don't raise spelling errors in the CI. Too many false positives. This is the kind of thing that you would catch with Vale locally though. Do you have it configured in VSCode?

Reply:
> We don't raise spelling errors in the CI. Too many false positives. This is the kind of thing that you would catch with Vale locally though. Do you have it configured in VSCode?

Yes.